### PR TITLE
Search result

### PR DESCRIPTION
**Pull request by mastemm**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Okey for me

File: backend/src/models/BandManager.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ class BandManager extends AbstractManager {
 
   findBand(id) {
     return this.connection.query(
-      `SELECT b.name,b.city,b.cachet,b.logo,a.firstname,a.lastname,a.avatar,r.label,mS.style FROM band AS b LEFT JOIN artist_has_band AS ahb ON b.id = ahb.band_id LEFT JOIN artist AS a ON ahb.artist_id=a.id LEFT JOIN role AS r on a.role_id=r.id LEFT JOIN bandMusicStyle AS bMS ON b.id=bMS.band_id LEFT JOIN musicStyle AS mS ON bMS.musicStyle_id=mS.id WHERE b.id = ?`,
+      `SELECT b.name,b.city,b.cachet,b.logo,a.firstname,a.lastname,a.avatar,a.id,r.label,mS.style FROM ${BandManager.table} as b inner join artist_has_band as ahb on b.id = ahb.band_id inner join artist as a on ahb.artist_id=a.id inner join role as r on a.role_id=r.id inner join bandMusicStyle as bMS on b.id=bMS.band_id
```
Comment (LuigiMenez) [inline backend/src/models/BandManager.js:15]: Try to maintain consistency in your queries: SELECT, INNER JOIN etc and not SELECT, inner join

**Review by LuigiMenez — APPROVED** [review]
Ok for me, Pay attention to query syntax and indentation.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**